### PR TITLE
Replace old URLs on change links for packaging sites

### PR DESCRIPTION
**Pull request by rajanimohan22**
Replace old URLs on change links for packaging sites
And also changes for 2179 for change link text

### CODE REVIEW

File: app/views/uniform/many.scala.html [hunk 1]
```diff
@@ -107,9 +107,6 @@ <h2 class="heading-small add-to-list" aria-hidden="true">@Messages(s"$key.list-c
                     </legend>
 
             }
-                @if(messages.isDefinedAt(s"$key.list-control.lead")) {
```
Comment (adamdye-hmrc) [inline app/views/uniform/many.scala.html]: Why has this been removed?

Comment (adamdye-hmrc) [reply app/views/uniform/many.scala.html]: I see why you have now, but the helper could be used by other pages in the future, probably best to just delete the relevant message here

Comment (rajanimohan22) [reply app/views/uniform/many.scala.html]: Removed the message but retained the tag as suggested.

Comment (adamdye-hmrc) [reply app/views/uniform/many.scala.html]: I meant to leave the If statement and span and message in the html. But delete the message we wanted to stop being shown, which is on line 825 of the messages file. That way we can still use this tag if we choose to define a message in future.

**Pull request merged**